n2y/export.py

```diff
@@ -58,6 +59,23 @@ def export_page(
 ):
     page_properties = _page_properties(page, pandoc_format, id_property, url_property, property_map)
     pandoc_ast = page.to_pandoc()
+    has_tables = pandoc_ast and any(isinstance(e, Table) for e in pandoc_ast[1])
+
+    if has_tables:
+        n_empty_headers = 0
+        for element in pandoc_ast[1]:
+            if isinstance(element, Table):
+                _, head = element[3]
+                n_header_rows = sum(1 for _, row in head)
+                if n_header_rows == 0:
+                    n_empty_headers += 1
+        if n_empty_headers:
+            logger.warning(
+                "%d table(s) will present empty headers to maintain Markdown spec",
+                n_empty_headers,
+                extra={"url": page.notion_url}
```

Comment:
Let's print out the URL directly (or does this already do this?). See other log messages to see how this is normally done.